core/src/main/java/org/hisp/dhis/android/core/mockwebserver/Dhis2MockServer.java

```diff
@@ -418,8 +412,4 @@ public String getBaseEndpoint() {
     public RecordedRequest takeRequest() throws InterruptedException {
         return server.takeRequest();
     }
-
-    public void addResponse(String method, String path, String responseName, int responseCode) {
```

Comment:
The methods `setDhis2Dispatcher()` and `addResponse()` are used in the Android Capture app for testing, they can't be removed.

Reply:
changed